### ISSUE TITLE
Manage Bower Packages: Context menu item appears for class libraries

### ISSUE THREAD

**Issue by scottaddie** [aspnet/Tooling#778]
I created a new ASP.NET Core project targeting .NET Core using the "StarterWeb" template in VS 2015 Update 3. I then added a class library (.NET Core) to the solution. When right-clicking the class library in Solution Explorer, the context menu appears with "Manage Bower Packages" as an option. Will anyone ever use Bower in anything other than a web project? In my opinion, this menu item shouldn't be present for class libraries.

**Comment by mlorbetske** [comment]
I believe we're working on a fix for this. @jodavis is that correct?

**Comment by jodavis** [comment]
Yes, that came up too late for the next preview, but we're working on a fix.
